src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(prevNum, curNum) {
+      return prevNum + curNum;
+    },
+
+    subtract(prevNum, curNum) {
+      return prevNum - curNum;
+    },
+
+    multiply(prevNum, curNum) {
+      return prevNum * curNum;
+    },
+
+    divide(prevNum, curNum) {
+      return prevNum / curNum;
```

Comment:
I think you need to change those parameters names to something like firstOperand and secondOperand
And also I think you should use arrow functions here